torchbench.py

```diff
@@ -177,13 +183,17 @@ def print_summary(cls):
             lines = "\n  ".join(map(str, v.most_common(50)))
             print(f"STATS {k}\n  {lines}")
 
+    @classmethod
+    def aot_summary(cls):
+        return [cls.totals["aot_autograd"]["total"], cls.totals["aot_autograd"]["ok"]]
+
 
-def output_csv(headers, row):
-    assert output_filename
-    existed = os.path.exists(output_filename)
+def output_csv(filename, headers, row):
```

Comment:
Is this change actually used?  What is motivation?

Reply:
I am using this later to write *_stats.csv file.

Reply:
Sounds good, I didn't see that